test/forms/inputs/cloudinary-file-input/cloudinary-uploader.test.js

```diff
@@ -350,21 +515,35 @@ describe('cloudinaryUploader', () => {
 
     expect.assertions(1)
 
-    return expect(upload(fileData, file)).rejects.toThrow()
+    await act(async () => {
+      await expect(upload(fileData, file)).rejects.toThrow()
+    })
```

Comment:
React 18 enforces `act` more strictly. I couldn't figure out a way to get rid of the `act` warning without using it directly

Reply:
The inner function doesn't have to be async. This works for me:

```
await act(() => expect(upload(fileData, file)).rejects.toThrow())
```

Reply:
Mostly just a syntax thing. You can either return the promise directly or await it in the body. I took your suggestion